### ISSUE TITLE
ArgumentError: Package BuildApp not found in current path

### ISSUE THREAD

**Issue by HamletFr** [NHDaly/ApplicationBuilder.jl#23]
Hi,

I tried to rebuild your example of the JuliaCon2018 and failed. I followed your installation instructions and the following two packages are installed:
```
[cc31668c] ApplicationBuilder v0.0.0 #master (https://github.com/NHDaly/ApplicationBuilder.jl.git)
[377060ff] ApplicationBuilderRuntimeUtils v0.0.0 #master (https://github.com/NHDaly/ApplicationBuilderRuntimeUtils.jl.git)
```

but if I run
`using ApplicationBuilder; using BuildApp;`
I get the following error:
```
ArgumentError: Package BuildApp not found in current path:
- Run `Pkg.add("BuildApp")` to install the BuildApp package.
```
What did I miss?

Thanks!

**Comment by ranjanan** [comment]
Are you on v1.0 or v0.7?

**Comment by HamletFr** [comment]
v0.7.0

**Comment by NHDaly** [comment]
Ah, sorry!! To get 0.7 to work I had to actually delete the BuildApp
module. I have it as a mental TODO to define an empty BuildApp module and
deprecate it, but I couldn't figure out how to expose one in the same way
as on 0.6... sorry! Sorry the 0.7/1.0 release is changing everything so
fast!

Just ignore the BuildApp, you don't need it!! Everything should just work
without it now. :)

Also, on 0.7, I've changed it to automatically do `change_dir_if_bundle`
for you, so you no longer need to import ApplicationBuilder into your
actual program. (and in fact you shouldn't)

**Comment by HamletFr** [comment]
Thanks for you long reply. If I just leave out the BuildApp, I geht the following error message:

```
julia> build_app_bundle("OurProject/src/project.jl")
ERROR: UndefVarError: build_app_bundle not defined
```

Thanks for helping!

**Comment by NHDaly** [comment]
Ohhhh I just realized you're still on matter.

Please checkout the juliav07 branch (I think that's what it's called) like
this:
] add ApplicationBuilder#juliav07

Sorry I'm on my phone right now still traveling, but I'm pretty sure that's
right... :D

**Comment by HamletFr** [comment]
Thanks - i completely oversaw that. I thought that the master branch is already 0.7

Thanks a lot for your help!

**Comment by NHDaly** [comment]
No problem. It's terribly documented. I think I added all the 0.7 like
within the last couple days, so I wouldn't be surprised if it's still not
fully working

**Comment by NHDaly** [comment]
Yeah! Is it like the third or fourth slide? That one is supposed to fail
for demonstration purposes.. haha sorry I should make that more clear in
the slides. Continue on reading further and it's explained.

**Comment by NHDaly** [comment]
Basically you need to wrap the code in julia_main. Read the section on what
ApplicationBuilder is doing under the hood.

**Comment by HamletFr** [comment]
Thanks. I will do that .. ! Have a nice Sunday.


Von: Nathan Daly <notifications@github.com>
Antworten an: "NHDaly/ApplicationBuilder.jl" <reply@reply.github.com>
Datum: Sonntag, 12. August 2018 um 12:25
An: "NHDaly/ApplicationBuilder.jl" <ApplicationBuilder.jl@noreply.github.com>
Cc: Tileman Conring <tileman@conring-frisius.de>, Author <author@noreply.github.com>
Betreff: Re: [NHDaly/ApplicationBuilder.jl] ArgumentError: Package BuildApp not found in current path (#23)

Basically you need to wrap the code in julia_main. Read the section on what
ApplicationBuilder is doing under the hood.

**Comment by NHDaly** [comment]
:) Hey @HamletFr, did you get everything taken care of? Let me know if you have any more questions! :)

I've finally merged the `julia_v07` branch into `master`, so this should all work just by default.

**Comment by HamletFr** [comment]
Yes, it now works perfectly. Thanks for all your support!
